sbi/utils/restriction_estimator.py

```diff
@@ -692,6 +692,11 @@ def sample(
                 max_sampling_batch_size=max_sampling_batch_size,
                 alternative_method="sample_with='sir'",
             )
+            # NOTE: This currently requires a float acceptance rate. A previous version
+            # of accept_reject_sample returned a float. In favour to batched sampling
+            # it now returns a tensor.
+            acceptance_rate = acceptance_rate.min().item()
```

Comment:
I see, this was moved out from `rejection.py`?

Reply:
Exactly. Also, that is why the min was returned previously (a similar note was in the sampler). I'm not sure if it is of any use here to not consider the min_acceptance_rate (hence I move the note here).